bl-kernel/abstract/dblist.class.php

```diff
@@ -94,9 +98,11 @@ public function remove($key)
 	// $args => 'name', 'oldkey', 'newKey', 'template'
 	public function edit($args)
 	{
-		if (isset($this->db[$args['newKey']])) {
+
+        if (isset($this->db[$args['newKey']])) {
```

Comment:
ok, but is not just remove the code, if a key exists for another category you are modifying a wrong category.